### ISSUE TITLE
Sync not starting and visualizer not showing anything

### ISSUE THREAD

**Issue by TheCurly** [BradyBrenot/huestacean#88]
I cannot get Huestacean to start syncing on my desktop pc that is connected to my TV. The bridge is connected and entertainment group is created. On my laptop it works fine. The desktop is an i5-2400 with integrated graphics running Windows 10. Any idea what could be wrong?

**Comment by TheCurly** [comment]
Any update on this issue? Screenbloom and Huedynamic both work fine but Screenbloom has much more latency and Huedynamic is a paid app.  I read about some known issues with Intel HD 3000 and 4000 in combination with Huestacean, but no information on Intel HD 2000.

**Comment by banksio** [comment]
@TheCurly Not the dev, but does it work when the TV is not connected? Do you have more than one graphics card?

**Comment by TheCurly** [comment]
I only have the TV connected with HDMI, I can try to use an old VGA screen and see if it does the same thing. Only one graphics card, the Intel HD 2000.

**Comment by banksio** [comment]
@TheCurly Do you have the correct drivers from Intel or your manufacturer's website? Sometimes the Windows Update drivers can break.

**Comment by TheCurly** [comment]
Intel does not have a driver for Win 10 for the HD graphics 2000 so I am using the windows update one. I could try if the Win 7/8 version from Intel would work.

**Comment by banksio** [comment]
@TheCurly Hmm, the fact Intel does not have a driver for Windows 10 for the HD 2000 might signal that it isn't compatible to the full extent with Windows 10, and Huestacean won't work. Feel free to try the older version though it won't hurt.

**Comment by TheCurly** [comment]
Older version didn't work either, unfortunately. I will try to find a windows 10 compatible video card to test.

**Comment by BradyBrenot** [comment]
Yeah these issues are really hard for me to figure anything out for, lacking the hardware and all. It does sound like some sort of compatibility issue RE: DXGI and that GPU.

**Comment by BradyBrenot** [comment]
Yeah, let me know how that works. I haven't dug into what they're doing yet. From briefly using it I would assume it's using the same APIs, so if it does work there's something odd going on.

**Comment by TheCurly** [comment]
The Hue sync app works, could it be the screen color grabbing part that does not work with my graphics card/driver?
